keps/sig-api-machinery/2340-Consistent-reads-from-cache/README.md

```diff
@@ -320,95 +351,49 @@ needed.
 
 ### Rollout, Upgrade and Rollback Planning
 
-<!--
-This section must be completed when targeting beta to a release.
--->
-
 ###### How can a rollout or rollback fail? Can it impact already running workloads?
 
-<!--
-Try to be as paranoid as possible - e.g., what if some components will restart
-mid-rollout?
-
-Be sure to consider highly-available clusters, where, for example,
-feature flags will be enabled on some API servers and not others during the
-rollout. Similarly, consider large clusters and how enablement/disablement
-will rollout across nodes.
--->
+API servers with feature enabled will experience different latency and throughput of LIST requests.
 
 ###### What specific metrics should inform a rollback?
 
-<!--
-What signals should users be paying attention to when the feature is young
-that might indicate a serious problem?
--->
+Users should check for increase if their apiserver latency and
```

Comment:
Any estimation what values of such metric we generally expect? I'm assuming it will be a histogram?

Reply:
I think 99%ile of apiserver_watch_cache_read_wait be below 100ms (period of fetching progress request).